src/main/webapp/app/lecture/lecture-attachments.component.ts

```diff
@@ -95,35 +94,56 @@ export class LectureAttachmentsComponent implements OnInit, OnDestroy {
         this.attachmentToBeCreated = newAttachment;
     }
 
+    /**
+     * If there is an attachment to save, it will be created or updated depending on its current state. The file will be automatically provided with the request.
+     */
     saveAttachment() {
```

Comment:
To avoid cases where some unecpected value gets retuned like below, we should specify this as a void method.
```suggestion
    saveAttachment(): void {
```